stories/Thing.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
If we are not using stories at all, we should remove this folder and the storybook dependency